### DISCUSSION TITLE
Parou de funcionar  ok eu de o fazer

### DISCUSSION THREAD

**Discussion by Adilio848** [anuraghazra/github-readme-stats#2334] [Q&A] [answered]
Meu aplicativo  parou de funcionar  oque  devo fazer

**Comment by rickstaa** [comment, marked as answer]
@Adilio848, can you please elaborate on the problems you are experiencing? The cards work for me:

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=Adilio848)](https://github.com/anuraghazra/github-readme-stats)
```

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=Adilio848)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=Adilio848)](https://github.com/anuraghazra/github-readme-stats)
```

[![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=Adilio848)](https://github.com/anuraghazra/github-readme-stats)
